databricks/koalas/frame.py

```diff
@@ -5201,25 +5190,6 @@ def replace(
         2     Thor  Mjolnir
         3     Hulk    Smash
 
-        Replacing value by specifying column
-
-        >>> df.replace('Mjolnir', 'Stormbuster', subset='weapon')
```

Comment:
Can we have an example that outputs the same since the dictionary is supported? I think that's the main reason why we're removing it here.

Reply:
@HyukjinKwon 
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.replace.html

According to the above document, I found that the `key` could be a `value` or a `column`. 
Koala seems to work the same. Should I change the documentation of doctest like below?


```python
>>> df = ks.DataFrame({"name": ['Ironman', 'Captain America', 'Thor', 'Hulk'],
        ...                    "weapon": ['Mark-45', 'Shield', 'Mjolnir', 'Smash']},
        ...                   columns=['name', 'weapon'])
        >>> df
                      name   weapon
        0          Ironman  Mark-45
        1  Captain America   Shield
        2             Thor  Mjolnir
        3             Hulk    Smash

        #  For a DataFrame a dict can specify that different values should be replaced in different column, 
        #  To use a dict in this way the value parameter should be None.

        >>> df.replace({'Mjolnir': 'Stormbuster'})
              name       weapon
        0   Rescue      Mark-45
        1  Hawkeye       Shield
        2     Thor  Stormbuster
        3     Hulk        Smash
       
        # For a DataFrame a dict can specify that different values should be replaced in different columns. 
        # The value parameter should not be None in this case.

        >>> df.replace({'weapon': 'Mjolnir'}, 'Stormbuster')
              name       weapon
        0   Rescue      Mark-45
        1  Hawkeye       Shield
        2     Thor  Stormbuster
        3     Hulk        Smash

        # For a DataFrame nested dictionaries. The value parameter should be None to use a nested dict in this way.

        >>> df.replace({'weapon': {'Mjolnir': 'Stormbuster'}})
              name       weapon
        0   Rescue      Mark-45
        1  Hawkeye       Shield
        2     Thor  Stormbuster
        3     Hulk        Smash
```